crates/biome_js_formatter/src/js/bindings/parameters.rs

```diff
@@ -214,7 +214,11 @@ pub(crate) fn should_hug_function_parameters(
 ) -> FormatResult<bool> {
     /// Returns true if the first parameter should be forced onto the same line as the `(` and `)` parentheses.
     /// See the `[ParameterLayout::Hug] documentation.
-    fn hug_formal_parameter(parameter: &self::AnyJsFormalParameter) -> FormatResult<bool> {
+    fn hug_formal_parameter(
+        parameter: &self::AnyJsFormalParameter,
+        from_arrow_function: bool,
+        has_parentheses: bool,
+    ) -> FormatResult<bool> {
```

Comment:
Since this function now accepts a new payload, it means that we need to update its documentation:
- explain what's new payload
- explain how the new payload affects the business logic

Reply:
updated~